spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewritePositionDeleteFiles.java

```diff
@@ -192,15 +192,15 @@ public void testNullTransform() throws Exception {
     testDanglingDelete(2);
   }
 
-  private <T> void testDanglingDelete() throws Exception {
+  private void testDanglingDelete() throws Exception {
     testDanglingDelete(NUM_DATA_FILES);
   }
 
-  private <T> void testDanglingDelete(int numDataFiles) throws Exception {
+  private void testDanglingDelete(int numDataFiles) throws Exception {
     Table table = Spark3Util.loadIcebergTable(spark, tableName);
 
     List<DataFile> dataFiles = dataFiles(table);
-    Assert.assertEquals(numDataFiles, dataFiles.size());
+    assertThat(numDataFiles).isEqualTo(dataFiles.size());
```

Comment:
nit, can probably use `assertThat(numDataFiles).hasSize(dataFiles.size())` as well for asserting on size of iterable (doing nullcheck as well internally)